src/pybind/mgr/orchestrator.py

```diff
@@ -323,6 +323,16 @@ def update_mons(self, num, hosts):
         """
         raise NotImplementedError()
 
+    def update_nfses(self, svc_id, num):
```

Comment:
can we just make use of `update_stateless_service` here, instead of adding a new method here?